packages/x-components/src/x-modules/recommendations/components/__tests__/recommendations.spec.ts

```diff
@@ -114,6 +114,27 @@ describe('testing recommendations component', () => {
 
     expect(isScopedSlotOverridden('custom-layout')).toBe(true);
   });
+
+  // eslint-disable-next-line max-len
+  it('renders at most the number of recommendations defined by `maxItemsToRender` prop', async () => {
+    const { wrapper, recommendations } = mountRecommendations();
+
+    const renderedRecommendations = (): WrapperArray<Vue> =>
```

Comment:
Like above, make the tests explicit about what are you testing. Use the `recommendations` array to do so, and remember to add the default case.